quinn/src/connection.rs

```diff
@@ -458,6 +459,10 @@ where
         self.0.lock("stats").inner.stats()
     }
 
+    pub fn cca(&self) -> Box<dyn Controller> {
```

Comment:
`congestion_controller`?  `cca` seems  only obvious for people who work on it. And what's the reason for making this one public - with an expensive copy instead of just returning a reference (which is problematic due to the lock)?

Reply:
this whole section was just for my debugging ease. any changes at all here are fine with me. my only goal was to be able to print out the internal variables once every few seconds while running a transfer.

Reply:
continued discussion here https://github.com/quinn-rs/quinn/pull/1151#discussion_r667590209